tests/test_cli_compile.py

```diff
@@ -759,6 +759,20 @@ def test_build_isolation_option(parse_requirements, runner, option, expected):
     assert parse_requirements.call_args.kwargs["options"].build_isolation is expected
 
 
+@mock.patch("piptools.scripts.compile.PyPIRepository")
+def test_forwarded_args(PyPIRepository, runner):
+    """
+    Test the forwarded cli args (--pip-args 'arg...') are passed to the pip command.
+    """
+    with open("requirements.in", "w"):
+        pass
+
+    pipc_args = ("--no-annotate", "--generate-hashes")
```

Comment:
```suggestion
    cli_args = ("--no-annotate", "--generate-hashes")
```
A nitpick. Feel free to disregard it :)

Reply:
:+1: